cohort_api_xblock/static/html/cohort_api_xblock.html

```diff
@@ -1,5 +0,0 @@
-<div class="cohort_api_xblock_block">
```

Comment:
will this unit be visible for the students ? so there will be  an empty component in the middle of the course content ...

Reply:
Yes, unfortunately, it's an empty component. But if you add it with another component, it's seamless